runner/host/fstab_test.go

```diff
@@ -9,29 +9,10 @@ import (
 
 	"github.com/hashicorp/hcdiag/op"
 
-	"github.com/hashicorp/hcdiag/runner"
 	"github.com/stretchr/testify/assert"
 )
 
-type mockShellRunner struct {
-	result map[string]any
-	status op.Status
-	err    error
-}
-
-func (m mockShellRunner) Run() op.Op {
-	return op.Op{
-		Result: m.result,
-		Status: m.status,
-		Error:  m.err,
-	}
-}
-
-func (m mockShellRunner) ID() string {
-	return ""
-}
-
-var _ runner.Runner = mockShellRunner{}
+//var _ runner.Runner = mockShellRunner{}
```

Comment:
Could we move this declaration over to host_test.go as well?

Reply:
Oh yeah I actually meant to remove it. What exactly is it doing because without it, the program still runs

Reply:
I had the same question when I started working on this project! This is a funky little trick to make sure we get a compile error if `mockShellRunner` ever stops implementing the runner.Runner interface. Basically, if we accidentally break something that's required to satisfy the `Runner` interface that the rest of hcdiag depends on, this will give a clear error when you try to `go build`.

Reply:
Nice!